stackable-certs/src/ca/k8s.rs

```diff
@@ -17,6 +17,9 @@ where
     #[snafu(display("failed to retrieve secret {secret:?}"))]
     GetSecret { source: kube::Error, secret: String },
 
+    #[snafu(display("invalid secret type, expected kubernetes.io/tls"))]
+    InvalidSecretType,
```

Comment:
nit (non-blocker): I think the Secret _type_ string shouldn't be written in two places. Either it should be a `const` or passed into the Snafu.

Reply:
Good catch! I will add a `const` for that.

Reply:
Fixed in 7a6fc12